.gitignore

```diff
@@ -1 +1,59 @@
-**/Pulumi.*.yaml
\ No newline at end of file
+# Pulumi
+**/Pulumi.*.yaml
+
+### Packer ###
```

Comment:
probably better keep the `.gitignore` specific to an example or a global one, doesn't make much sense to have multiple ones

Reply:
i think keep the global changes :+1: